scss/neue.scss

```diff
@@ -49,6 +49,7 @@
 @import "_components/spinner";
 @import "_components/video";
 @import "_components/waypoints";
+@import "_components/avatar";
```

Comment:
This is a kinda silly nitpick, but can you move this to the top of the list here & in the pattern library file so it stays alphabetical? :grimacing: